pdsc/segment.py

```diff
@@ -73,7 +73,7 @@ class SegmentTree(object):
     observation segments within some radius of a query point
     """
 
-    def __init__(self, segments, verbose=True):
+    def __init__(self, segments, verbose=True, body_radius=MARS_RADIUS_M):
```

Comment:
Would you add a doc string comment for the new param?

Reply:
added